### DISCUSSION TITLE
Theia IDE 1.56 preview testing

### DISCUSSION THREAD

**Discussion by sgraband** [eclipse-theia/theia#14564] [Release Announcements] [answered]
The new version 1.56.0 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.56.0/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.56.0/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.56.0/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference updates.channel to preview. Please respond here when you were able to test the preview without finding blockers, by commenting with a ✔️. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by JonasHelming** [comment]
@sgraband I believe, we are missing the HuggingFace Provider in the Theia IDE

**Reply by sgraband** [reply]
1.56.1 is available on the preview channel, which includes the HuggingFace Provider.

**Reply by JonasHelming** [reply]
HuggingFace works well for me :heavy_check_mark:

**Comment by tsmaeder** [comment]
Can we still get a +/- 1 from all testers by tonight, please? Even if the HuggingFace-Provider does not work at all, we can still release a version, since it's no worse than before.

**Reply by sgraband** [reply]
:heavy_check_mark: Good from my side

**Comment by rschnekenbu** [comment]
✔️ +1 for me. I did not catch new issues, especially on file related operations. I could not test much of the AI feature.

**Comment by tsmaeder** [comment]
+1 from me, as well.

**Comment by jfaltermeier** [comment]
+1

**Comment by tsmaeder** [comment]
We're good to go.

**Reply by sgraband** [reply, marked as answer]
1.56.1 is promoted to the stable channel

**Reply by rschnekenbu** [reply]
Thanks for the release :heart:

**Reply by JonasHelming** [reply]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/12/06/eclipse-theia-1-56-release-news-and-noteworthy/)
